package.json

```diff
@@ -27,7 +27,7 @@
     "build": "yarn workspace as-serde-transform build",
     "checks": "yarn workspaces run checks",
     "fmt": "prettier -w './**/*.ts' --config ./.prettierrc.cjs",
-    "fmt-check": "prettier -w './**/*.ts' --config ./.prettierrc.c`js --check",
+    "fmt-check": "prettier -w './**/*.ts' --config ./.prettierrc.cjs --check",
```

Comment:
it's better to enable the `fmt-check` in the CI

Reply:
No, it works not good for AS lang. It will always return non-zero error code. But `yarn lint` will be more friendly.